app/controllers/map_controller.rb

```diff
@@ -156,10 +156,28 @@ def set_labels
     }
   end
 
-  def pt_bounds
-    [
+  def default_bounds
+    pt_bounds = [
```

Comment:
What do you think of doing something like:

```ruby
  BOUNDS = {
    "pt" => [
      [-8.693305501609501, 41.11531424472605],
      [-8.520026252042584, 41.17823491872437]
    ],
    "es" => [
      [-3.941566736714776,40.328785315750025],
      [-3.4482048268615415,40.515118259540344]
    ],
    "sk" => [
      [12.612613950622432,45.377461677566686],
      [16.88953848960196,46.848256073855225]
    ]
  }

  def default_bounds
    bounds.fetch(I18n.locale.to_s) { bounds["pt"] }
  end
```

Reply:
i think that's elegant asf 😄 , I'm just discovering ruby so pardon my french 😛 .

Reply:
Hugo is a great mentor =D